### PR TITLE
Add script covering case when Navi.IsReady gets no responce from HMI …

### PR DESCRIPTION
**Pull request by vvvakulenko**
…and Navi.AlertManeuver (UNSUPPORTED_RESOURCE), TTS.Speak (WARNINGS)
Script is a part of TS #666

### CODE REVIEW

**Comment by vvvakulenko** [comment]
@istoimenova Please review

File: test_scripts/API/AlertManeuver/015_ATF_Navi_IsReady_no_resp_AlertManeuver_Navi_AlertMan_UNSUP_RES.lua [hunk 1]
```diff
@@ -0,0 +1,174 @@
+---------------------------------------------------------------------------------------------
```
Comment (vvvakulenko) [inline test_scripts/API/AlertManeuver/015_ATF_Navi_IsReady_no_resp_AlertManeuver_Navi_AlertMan_UNSUP_RES.lua]: @istoimenova , updated in commit 713c3db0ad5ad1aa14f1f4c9b5d386c5d7950b4c

File: test_scripts/API/AlertManeuver/015_ATF_Navi_IsReady_no_resp_AlertManeuver_Navi_AlertMan_UNSUP_RES.lua [hunk 2]
```diff
@@ -0,0 +1,174 @@
...
+--[[ General Settings for configuration ]]
+Test = require('user_modules/connecttest_initHMI')
+require('user_modules/AppTypes')
+
+--[[ Preconditions ]]
+commonFunctions:newTestCasesGroup("Preconditions")
+
+function Test:Precondition_InitHMI_OnReady()
+  testCasesForNavi_IsReady.InitHMI_onReady_without_Navi_IsReady(self, 1)
+  EXPECT_HMICALL("Navi.IsReady")
```
Comment (vvvakulenko) [inline test_scripts/API/AlertManeuver/015_ATF_Navi_IsReady_no_resp_AlertManeuver_Navi_AlertMan_UNSUP_RES.lua]: @istoimenova , updated in commit 713c3db0ad5ad1aa14f1f4c9b5d386c5d7950b4c

File: test_scripts/API/AlertManeuver/015_ATF_Navi_IsReady_no_resp_AlertManeuver_Navi_AlertMan_UNSUP_RES.lua [hunk 3]
```diff
@@ -0,0 +1,174 @@
...
+  -- Do not send HMI response of Navi.IsReady
```
Comment (vvvakulenko) [inline test_scripts/API/AlertManeuver/015_ATF_Navi_IsReady_no_resp_AlertManeuver_Navi_AlertMan_UNSUP_RES.lua]: @istoimenova , updated in commit 713c3db0ad5ad1aa14f1f4c9b5d386c5d7950b4c

**Comment by vvvakulenko** [comment]
@istoimenova , Please see updated script on commit 713c3db0ad5ad1aa14f1f4c9b5d386c5d7950b4c

File: user_modules/IsReady_Template/testCasesForNavi_IsReady.lua [hunk 4]
```diff
@@ -0,0 +1,273 @@
...
+    { attenuatedSupported = true })
+  ExpectRequest("BasicCommunication.GetSystemInfo", false,
+    {
+      ccpu_version = "ccpu_version",
+      language = "EN-US",
+      wersCountryCode = "wersCountryCode"
+    })
+  ExpectRequest("UI.GetLanguage", true, { language = "EN-US" })
+  :Times(exp_occur)
+  :Timeout(20000)
```
Comment (istoimenova) [inline user_modules/IsReady_Template/testCasesForNavi_IsReady.lua]: @vvvakulenko, as UI.IsReady will come, expectation for UI.GetLanguage should be removed.
You should have only "ExpectRequest("UI.GetLanguage", true, { language = "EN-US" })", so please remove line 37-38

File: user_modules/IsReady_Template/testCasesForNavi_IsReady.lua [hunk 5]
```diff
@@ -0,0 +1,273 @@
...
+        "NO-NO","NL-BE","EL-GR","HU-HU","FI-FI","SK-SK" }
+    })
+  ExpectRequest("UI.GetSupportedLanguages", true, {
+      languages = {
+        "EN-US","ES-MX","FR-CA","DE-DE","ES-ES","EN-GB","RU-RU",
+        "TR-TR","PL-PL","FR-FR","IT-IT","SV-SE","PT-PT","NL-NL",
+        "ZH-TW","JA-JP","AR-SA","KO-KR","PT-BR","CS-CZ","DA-DK",
+        "NO-NO","NL-BE","EL-GR","HU-HU","FI-FI","SK-SK" }
+    })
+  :Times(exp_occur)
```
Comment (istoimenova) [inline user_modules/IsReady_Template/testCasesForNavi_IsReady.lua]: @vvvakulenko, as UI.IsReady will come, expectation for UI.GetLanguage should be removed.
You should have only "ExpectRequest("UI.GetSupportedLanguages", true, {.....})", so please remove lines 68-69

File: test_scripts/API/AlertManeuver/015_ATF_Navi_IsReady_no_resp_AlertManeuver_Navi_AlertMan_UNSUP_RES.lua [hunk 6]
```diff
@@ -0,0 +1,174 @@
...
+
+--[[ General Settings for configuration ]]
+Test = require('user_modules/connecttest_initHMI')
+require('user_modules/AppTypes')
+
+--[[ Preconditions ]]
+commonFunctions:newTestCasesGroup("Preconditions")
+
+function Test:Precondition_InitHMI_OnReady()
+  testCasesForNavi_IsReady.InitHMI_onReady_without_Navigation_IsReady(self, 1)
```
Comment (istoimenova) [inline test_scripts/API/AlertManeuver/015_ATF_Navi_IsReady_no_resp_AlertManeuver_Navi_AlertMan_UNSUP_RES.lua]: @vvvakulenko, the function that you are using is with different name in file:
testCasesForNavi_IsReady.lua. There it is named InitHMI_onReady_without_Navi_IsReady()

**Comment by vvvakulenko** [comment]
@istoimenova  Please review updates

File: test_scripts/API/AlertManeuver/015_ATF_Navi_IsReady_no_resp_AlertManeuver_Navi_AlertMan_UNSUP_RES.lua [hunk 7]
```diff
@@ -0,0 +1,174 @@
...
+        image = { imageType = "DYNAMIC"},
+        isHighlighted = true,
+        softButtonID = 821,
+        systemAction = "DEFAULT_ACTION",
+    }},
+    appID = self.applications[config.application1.registerAppInterfaceParams.appName]
+  })
+  :ValidIf(function(_,data)
+    local value_Icon = storagePath .. "icon.png"
+    if (string.match(data.params.softButtons[1].image.value, "%S*" .. "("..string.sub(storagePath, 2).."icon.png)" .. "$") == nil ) then
```
Comment (istoimenova) [inline test_scripts/API/AlertManeuver/015_ATF_Navi_IsReady_no_resp_AlertManeuver_Navi_AlertMan_UNSUP_RES.lua]: @vvvakulenko, based on CI problem with verification of dynamic image, please add:
(data.params.softButtons[1].image.value ~= value_Icon ) in 'if' condition.

File: test_scripts/API/AlertManeuver/015_ATF_Navi_IsReady_no_resp_AlertManeuver_Navi_AlertMan_UNSUP_RES.lua [hunk 8]
```diff
@@ -0,0 +1,174 @@
...
+      print("\27[31m value of softButtons.image is WRONG. Expected: ~".. value_Icon .. "; Real: " .. data.params.softButtons[1].image.value .. "\27[0m")
```
Comment (istoimenova) [inline test_scripts/API/AlertManeuver/015_ATF_Navi_IsReady_no_resp_AlertManeuver_Navi_AlertMan_UNSUP_RES.lua]: @vvvakulenko, please remove '~' from description.

**Pull request merged**